src/screens/ProfileTwoFactorAuthScreen/index.tsx

```diff
@@ -132,11 +132,11 @@ class ToggleTwoFactorAuthComponent extends React.Component<Props, State> {
                 <div className="row m-0 pg-profile-two-factor-auth__body">
                     <div className="col-12 pl-2 pg-profile-two-factor-auth__body--text d-inline-block">
                         <div className="row col-12 pg-profile-two-factor-auth__body--text--group">
-                            <div className="col-12 col-md-8 col-sm-7">
+                            <div className="col-12 col-sm-5">
```

Comment:
could you please adjust the field to that it fit max-content width. In english language the field should stay the same length as it was previously

Reply:
set 2FA input width by placeholder text's width.